src/webview/src/App.js

```diff
@@ -222,6 +223,9 @@ export default class App extends Component {
       this.state.fbPath = [{ title: this.state.title, effectIndex: 0 }]
     }
 
+    // Ensure summary dialog is initially disabled
+    this.setState({ showSummaryDialog: false })
```

Comment:
I think that in constructor (and only there) you should modify this.state directly. e.g. 
```
this.state['showSummaryDialog']=false;
```

Reply:
I agree.